modules/engine/src/index.ts

```diff
@@ -1199,15 +1115,15 @@ export class VectorEngine implements IVectorEngine {
       "both",
     );
     if (sigRes.isError) {
-      return sendResponseToCounterparty(RestoreError.reasons.InvalidSignatures, {
+      return generateRestoreError(RestoreError.reasons.InvalidSignatures, {
         recoveryError: sigRes.getError().message,
       });
     }
 
     // Verify transfers match merkleRoot
     const { root } = generateMerkleTreeData(activeTransfers);
     if (root !== channel.merkleRoot) {
-      return sendResponseToCounterparty(RestoreError.reasons.InvalidMerkleRoot, {
+      return generateRestoreError(RestoreError.reasons.InvalidMerkleRoot, {
```

Comment:
The functions `syncDisputes` and `restoreState` are `private` and unused. Either this is a bug or they should be deleted.

Reply:
All functions in the engine are marked as private -- you access them via the rpc methods